src/compas_fab/robots/constraints.py

```diff
@@ -183,37 +239,56 @@ def __init__(self, joint_name, value, tolerance_above=0., tolerance_below=0., we
         self.tolerance_below = abs(tolerance_below)
 
     def scale(self, scale_factor):
+        """Scale (multiply) the constraint with a factor.
+
+        Parameters
+        ----------
+        scale_factor : :class:`float`
+            Factor used to multiply the joint value and tolerance bounds with.
+        """
         self.value *= scale_factor
         self.tolerance_above *= scale_factor
         self.tolerance_below *= scale_factor
 
     def __repr__(self):
+        """Printable representation of :class:`JointConstraint`."""
         return "JointConstraint('{0}', {1}, {2}, {3}, {4})".format(self.joint_name, self.value, self.tolerance_above, self.tolerance_below, self.weight)
 
     def copy(self):
+        """Create a copy of this :class:`JointConstraint`.
+
+        Returns
+        -------
+        :class:`JointConstraint`
+        """
         cls = type(self)
         return cls(self.joint_name, self.value, self.tolerance_above, self.tolerance_below, self.weight)
 
 
 class OrientationConstraint(Constraint):
-    """Constrains a link to be within a certain orientation.
+    r"""Constrains a link to be within a certain orientation.
 
     Attributes
     ----------
-    link_name: string
+    link_name : :class:`str`
         The name of the link this contraint refers to.
-    quaternion: list of float
+    quaternion : :class:`list` of :class:`float`
         The desired orientation of the link specified by a quaternion in the
-        order of [w, x, y, z].
-    tolerances: list of float, optional
-        Error tolerances ti for each of the frame's axes. The respective
-        bound to be achieved is [ai - ti, ai + ti]. Defaults to [0.01, 0.01, 0.01].
-    weight: float, optional
+        order of ``[w, x, y, z]``.
+    tolerances : :class:`list` of :class:`float`, optional
+        Error tolerances t\ :sub:`i` for each of the frame's axes. If only one
+        value is passed it will be used for all 3 axes. The respective bound to
+        be achieved is ``[ai - ti, ai + ti]``. Defaults to
```

Comment:
what about 
> The respective bound to be achieved is :math:`(a_{i} - t_{i}, a_{i} + t_{i})`
or  even ":math:`a_{i} \pm t_{i}`" (assuming standard latex works here)?

Reply:
Thanks for reviewing!

Maybe option 1 would be best, to make it easier to read when looking at the source code. What do you think?

Reply:
sounds good

Reply:
c43fb9b